### ISSUE TITLE
Add hourly forecast section

### ISSUE THREAD

**Issue by Makin-Things** [Makin-Things/platinum-weather-card#59]
It would be great to have an hourly forecast section for the card. I am not sure how to best format it but it should be able to deal with up to 72 hours (3 days) of data (temp, wind, rain etc).

**Comment by enriquerollon** [comment]
If it's going to cover up to 72 hours, it would be nice to allow horizontal (or vertical) scrolling. Although I don't know if it's technically possible.

Something similar to the ios weather app:
![image](https://user-images.githubusercontent.com/85040979/196426983-89ec695d-6359-42b0-be1e-0f0f44cc534a.png)

It is also an idea to integrate warning section.

**Comment by oneseventhree** [comment]
+1 for this! I would love it if it followed the standard weather card but just with the icons of this card:
![image](https://user-images.githubusercontent.com/15043282/200729180-32a74e76-b347-4bdd-97b0-cd6427dab7d8.png)

**Comment by The8bitRebel** [comment]
Yes please! Google weather has a daily forecast as well as a scrollable hourly forecast for each day. It would be incredible if this could be implemented!
![IMG_0A42441DE983-1](https://user-images.githubusercontent.com/102759263/208792593-400b2f2a-67d2-45e5-b172-b7c89ee74211.jpeg)

**Comment by badewanne1234** [comment]
+1 for this! :)

**Comment by abenedet78** [comment]
+1 for this! :)

**Comment by oleg-d** [comment]
another +1

**Comment by glibg10b** [comment]
Also these from https://github.com/decompil3d/lovelace-hourly-weather (they can be set via templates, which allows a lot of freedom)

<details><summary>Click to expand</summary>

| Name                             | Type             | Requirement  | Description                                                                                                                                                     | Default             |
|----------------------------------|------------------|--------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------|
| `name`                           | string           | **Optional** | Card name (set to `null` to hide)                                                                                                                               | `Hourly Weather`    |
| `num_segments`                   | number           | **Optional** | Number of forecast segments to show (integer >= 1)                                                                                                              | `12`                |
| `offset`                         | number           | **Optional** | Number of forecast segments to offset from start                                                                                                                | `0`                 |
| `label_spacing`                  | number           | **Optional** | Space between time/temperature labels (integer >= 1)                                                                                                            | `2`                 |
</details>
